src/components/tabs.js

```diff
@@ -46,6 +47,13 @@
       }
     }, [isDev, currentTab]);
 
+    useEffect(() => {
+      // Manually trigger updateIndicator() to show active tab on page load (TECHSUP-8534)
```

Comment:
Remove comment

Reply:
This comment was suggested by your team mate so that we can find it for future references, but I'm fine with removing it.